core/src/test/java/io/questdb/cairo/TableWriterTest.java

```diff
@@ -1592,28 +1602,7 @@ public int findNext(long findPtr) {
                 }
                 return super.findNext(findPtr);
             }
-        }, true);
-    }
-
-    @Test
-    public void testDayPartitionTruncateError() throws Exception {
```

Comment:
removed these tests because we no longer truncate columns in partitioned table, we remove partitions instead